src/test/ui/typeck/struct-enum-wrong-args.rs

```diff
@@ -0,0 +1,13 @@
+struct Wrapper(i32);
```

Comment:
Could you add a reference to the issue as a comment? This will make it easier to understand what the problem was and what this is testing.

```suggestion
// Regression test of #86481.

struct Wrapper(i32);
```